src/components/SQForm/SQFormDateTimePicker.tsx

```diff
@@ -1,10 +1,27 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import moment from 'moment';
 import Grid from '@material-ui/core/Grid';
 import TextField from '@material-ui/core/TextField';
-import {DateTimePicker} from '@material-ui/pickers';
+import {DateTimePicker, BaseDateTimePickerProps} from '@material-ui/pickers';
 import {ClickAwayListener, makeStyles} from '@material-ui/core';
 import {useForm} from './useForm';
+import BaseFieldProps from '../../types/BaseFieldProps';
+import {ParsableDate} from '@material-ui/pickers/constants/prop-types';
+
+interface SQFormDateTimePickerProps extends BaseFieldProps {
+  /** Disabled property to disable the input if true */
+  isDisabled?: boolean;
+  /** Required property used to highlight input and label if not fulfilled */
+  isRequired?: boolean;
```

Comment:
I think we've gone away with this prop on our components since it should be automatic now

Reply:
quite right, removing it.